Sources/Nimble/Adapters/NimbleXCTestHandler.swift

```diff
@@ -78,7 +78,16 @@ public func recordFailure(_ message: String, location: SourceLocation) {
 #else
     if let testCase = CurrentTestCaseTracker.sharedInstance.currentTestCase {
         let line = Int(location.line)
+#if swift(>=5.3)
+        let location = XCTSourceCodeLocation(filePath: location.file, lineNumber: line)
+        let sourceCodeContext = XCTSourceCodeContext(location: location)
+        let issue = XCTIssue(type: .assertionFailure, compactDescription: message,
+                             detailedDescription: message, sourceCodeContext: sourceCodeContext,
+                             associatedError: nil, attachments: [])
```

Comment:
```suggestion
        let issue = XCTIssue(type: .assertionFailure, compactDescription: message, sourceCodeContext: sourceCodeContext)
```

- You can omit default arguments
- I think there is no need to pass the same `message` to both `compactDescription` and `detailedDescription`

https://developer.apple.com/documentation/xctest/xctissue

> var compactDescription: String
> A concise description of the issue with no transient data, suitable for use in test run summaries and results aggregation across multiple test runs.

> var detailedDescription: String?
> A detailed description of the issue that may include transient data, such as numbers, object identifiers, and timestamps, to help diagnose the issue.